zio-http/src/test/scala/zhttp/service/ClientContentLengthSpec.scala

```diff
@@ -1,12 +1,11 @@
 package zhttp.service
 
-import zhttp.http.{Http, Response}
+import zhttp.http.{Http, Response, _}
```

Comment:
Cant understand why equivalent changes in  import  (from point of view of idea) broke build ...